src/display-layer.coffee

```diff
@@ -126,6 +127,9 @@ class DisplayLayer
       "\r\n": @invisibles.cr + @invisibles.eol
     }
 
+  recomputeScreenLinesIfNeeded: ->
+    return unless @propertiesHaveChanged
+    @propertiesHaveChanged = false
```

Comment:
Super minor stylistic comment here. 🎨 What do you think about changing this into:

```coffee
if @propertiesHaveChanged
  @propertiesHaveChanged = false
  # etc. etc.
```

Feel free to ignore this if you don't think it helps readability. 👍